<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boarding Actions App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>Welcome to BA list builder v 0.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="close_app_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Close App</string>
    </property>
   </widget>
   <widget class="QToolButton" name="new_list_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>New List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewList"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewList">
   <property name="text">
    <string>NEW LIST</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
